src/components/Tabs/Tabs.tsx

```diff
@@ -74,10 +74,6 @@ export interface Props {
    * Function passed down from higher level component into Tabs
    */
   tabsOnClick?: () => void;
-  /**
-   * Tabs rendered on a dark backgorund
-   */
-  inverted?: boolean;
   /**
    * Overflow variants
```

Comment:
Ugh, looks like there are a few props in here that aren't being used at all—`overflow`, `variant`, `radioLegendPosition`, `hideLegend`, `radioLegendPosition`, `tabsOnClick`... https://app.shortcut.com/czi-edu/story/216176/remove-unused-props-from-tabs

Reply:
@dierat i can pick up that prop clean up ticket since i am in here (small follow-up PR)

Reply:
Thanks! Here's the `TimelineNav` equivalent if you're interested: https://app.shortcut.com/czi-edu/story/216177/remove-unused-props-from-timelinenav